cli/command/stack/deploy.go

```diff
@@ -30,7 +30,7 @@ func newDeployCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "deploy [OPTIONS] STACK",
-		Aliases: []string{"up"},
+		Aliases: []string{"up", "docker deploy"},
```

Comment:
Nope, this alias is `docker stack up`, and that's where the problem lies. If we make this `docker stack up`, it stops working as an actual alias, `docker stack up` would be an "error not a subcommand".

If we want it to print `docker stack up` in the help test we need to change `cobra` to print the full command, not just the relative alias. 

We can do that if we want, I don't think it will be too bad.